Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:focal AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+ADD requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:lts-slim AS yarn-dependencies
```

Comment:
Are we sure `node:lts-slim` will still pull `node 16` whenever there is a new active LTS release of node? I would, if possible explicitly pull `node 16` just to avoid any surprises.

Reply:
At this point it is pinned at [16.13](https://github.com/nodejs/docker-node/blob/d146f71011bc87d739c9ed3e2820726af7d75417/16/buster-slim/Dockerfile) I think it's a good idea, because this will update to LTS and not make us update manually every once in a while. If for some reason it fails, the build most likely will fail and we can fix it.

Reply:
I guess If we can make renovate notify us when an LTS update is available we can pin the exact version

Reply:
I'd rather pin this to `16` so it's really clear what version of node we are using to build our assets. It could also be the case that `node:lts-slim` upgrades and breaks things silently and we would not be aware. Plus it's not that hard to upgrade node when we want to move to a different version.

Reply:
Good point, I specified the version `16`, renovate will probably suggest to upgrade to 17 but we will just ignore it.
I tried to specify in package.json the option `engines` property to LTS but there is no such value option, it only supports version condition `16`, `16.X`, `>=16`, etc...